django/db/backends/sqlite3/introspection.py

```diff
@@ -192,9 +192,9 @@ def get_primary_key_column(self, cursor, table_name):
         fields_sql = create_sql[create_sql.index('(') + 1:create_sql.rindex(')')]
         for field_desc in fields_sql.split(','):
             field_desc = field_desc.strip()
-            m = re.search('"(.*)".*PRIMARY KEY( AUTOINCREMENT)?', field_desc)
+            m = re.match('"?([\w-]+)"?.*PRIMARY KEY.*', field_desc)
```

Comment:
We are using `re.match` because it matches from the beginning of the string and we want to return the first capture group. Quotation is optional and the wildcard expression at the end covers `AUTOINCREMENT` or `NOT NULL` usage.